mxcube3/__init__.py

```diff
@@ -67,10 +67,10 @@ def exception_handler(e):
     import logging_handler
     #root_logger = logging.getLogger()
     #root_logger.setLevel(logging.DEBUG)
-    hwr_logger = logging.getLogger("HWR")
+    #hwr_logger = logging.getLogger("HWR")
+    hwr_logger = logging.getLogger("user_level_log")
```

Comment:
Sorry to come back on this now... Can you explain why there is a huge improvement
in speed by using this logger instead of HWR ?
I understand there is much less traffic on this one, but it's not the same : one is a general purpose
log (HWR), the other one is just for showing some messages to the user (user_level_log).
With this change we don't process a lot of messages that were displayed in MXCuBE 2 in the
System tab. Is it what we want ?
There is still the idea to use Sentry for example for storing HWR log, then in this case we don't need all the log messages in MXCuBE 3... However I didn't install it here yet, it's a more complex task than just having log messages in the application. Did you give it a try ?